packages/@ember/-internals/routing/lib/services/router.ts

```diff
@@ -244,12 +244,33 @@ export default class RouterService extends Service {
   }
 
   /**
-     Determines whether a route is active.
+     Returns `true` if `routeName/models/queryParams` is the active route, where `models` and `queryParams` are optional.
+     See [model](api/ember/release/classes/Route/methods/model?anchor=model) and
+     [queryParams](/api/ember/3.7/classes/Route/properties/queryParams?anchor=queryParams) for more information about these arguments.
+
+     In the following example, isActive will return `true` if the current route is `/posts`.
+     ```app/components/posts.js
+     import Component from '@glimmer/component';
+     import { inject as service } from '@ember/service';
+
+     export default class extends Component {
+       @service router;
+
+       displayComments() {
+         return this.get('router').isActive('posts');
+       }
+     });
+     ```
+
+     Usage example  with a model:
+     ```javascript return this.get('router').isActive('posts', post.id)```
```

Comment:
```suggestion
     
     ```javascript
     return this.get('router').isActive('posts', post.id)
     ```
```

Reply:
We also try to make the code samples as complete as possible. Can you add some more context to it so it's easier to understand where it goes?